monitoring/uss_qualifier/Makefile

```diff
@@ -1,9 +1,19 @@
+.PHONY: ci
+ci: install_dependencies test
+
+.PHONY: install_dependencies
+install_dependencies:
+	pip install black
 
 .PHONY: test
 test: lint
 	./scripts/test_docker_fully_mocked.sh
 
 .PHONY: lint
 lint:
+	black --check .
 	find . -name '*.sh' | xargs docker run --rm -v $(CURDIR):/code -w /code koalaman/shellcheck
 
+.PHONY: format
```

Comment:
Can we document that developers should use this to fix issues detected by the `lint` target above in the CI?  (assuming that understanding is correct)

Reply:
Please let me know if documenting in [DEVELOPMENT.md](https://github.com/interuss/dss/blob/9f3227d750658e7ecf81235421d44e9f47acb652/DEVELOPMENT.md?plain=1#L18-L20) and [printing an error message](https://github.com/interuss/dss/blob/9f3227d750658e7ecf81235421d44e9f47acb652/monitoring/uss_qualifier/Makefile#L7) address your remark ?

Reply:
Yep, that looks great; thanks!